pkg/tools/etcd_helper_watch_test.go

```diff
@@ -205,7 +217,7 @@ func TestWatchEtcdError(t *testing.T) {
 	fakeClient := NewFakeEtcdClient(t)
 	fakeClient.expectNotFoundGetSet["/some/key"] = struct{}{}
 	fakeClient.WatchImmediateError = fmt.Errorf("immediate error")
-	h := EtcdHelper{fakeClient, codec, versioner}
+	h := NewEtcdHelper(fakeClient, codec)
```

Comment:
That's true, but on the other hand we want test setup that is consistent with prod.